apps/che/src/main/fabric8/deployment.yml

```diff
@@ -144,13 +144,17 @@ spec:
         imagePullPolicy: "IfNotPresent"
         name: che
         readinessProbe:
-          tcpSocket:
+          httpGet:
+            path: /api/system/state
             port: 8080
-          initialDelaySeconds: 20
+            scheme: HTTP
+          initialDelaySeconds: 60
```

Comment:
@mlabuda what figures would you propose for liveness / readiness (server startup takes ~ 40 - 60 seconds) ? I actually thought that that timeout is just number of seconds after which the probe times outs and does not have anything in common with periodic checks